components/main/layout/navbar.tsx

```diff
@@ -25,14 +25,20 @@ export type MainNavbarGroup = {
   items: NavLink[];
 };
 
+export type MainNavbarItemTexts = {
```

Comment:
I'm really unsure what to call this. Is "text" a good way to describe it?

Reply:
I would probably not create a new type for this, I think it would be fine to just have it inline on `MainNavbarProps`, no need to export it ☺️